### PR TITLE
Added scala-implicit-browser

### PR DESCRIPTION
**Pull request by fikovnik**

### CODE REVIEW

File: analysis/scala-implicit-browser/app.R [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+          choices=NULL
+        ),
+        h4("Filter"),
+        checkboxGroupInput(
+          inputId="filter_kind",
+          label=h5("by kind")
+        ),
+        checkboxGroupInput(
+          inputId="filter_srcset",
+          label=h5("by srcset")
```
Comment (blorente) [inline analysis/scala-implicit-browser/app.R]: Perhaps a more descriptive name (just `source group`, or whatever. just full words), as a small detail.

File: analysis/scala-implicit-browser/app.R [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+      NULL
+    } else {
+      model$nodes %>%
+        filter(node_type == NODE_FUN) %>%
+        select(srcset) %>%
+        distinct() %>%
+        .$srcset
+    }
+
+    updateCheckboxGroupInput(
```
Comment (blorente) [inline analysis/scala-implicit-browser/app.R]: Slight bug, which may totally come from the data in the database being ill-formatted. If that's the case, we don't have to worry.

wonky srcsets

File: analysis/scala-implicit-browser/app.R (hunk 1 above)
Comment (fikovnik) [reply analysis/scala-implicit-browser/app.R]: I will call it `source directory` - that seems to be sbt terminology.

File: analysis/scala-implicit-browser/app.R (hunk 2 above)
Comment (fikovnik) [reply analysis/scala-implicit-browser/app.R]: Good point, I was wondering when it will show up :-)

Comment (fikovnik) [reply analysis/scala-implicit-browser/app.R]: Fixed in f693744

**Pull request merged**